### PR TITLE
Feat/gigalayer logo

### PR DESCRIPTION
**Pull request by ladaposamuel**

### CODE REVIEW

File: logos/gigalayer/gigalayer.svg [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Odusanya) [inline logos/gigalayer/gigalayer.svg]: Hi @ladaposamuel, thank you for taking time to contribute to this. 

Please can you run your files through [SVGOMG](https://jakearchibald.github.io/svgomg/) and an image optimizer, as recommended in the [contributing.md](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md) file.

File: js/logos.json [hunk 2]
```diff
@@ -322,4 +322,8 @@
 	"title": "Globacom Limited",
 	"filename": "glo",
 	"category": "Telecommunication / ISP"
-}]
+}, {
+	"title": "Gigalayer",
+	"filename": "gigalayer",
+	"category": "Hosting Company"
```
Comment (Odusanya) [inline js/logos.json]: Hey @ladaposamuel, can we categorize Gigalayer as a `Software / ICT` company? `Hosting company` is not immediately clear to a non-tech savvy person.

Comment (ladaposamuel) [reply js/logos.json]: Thank you,
Changed it to `Software / ICT`

File: logos/gigalayer/gigalayer.svg (hunk 1 above)
Comment (ladaposamuel) [reply logos/gigalayer/gigalayer.svg]: I have done this thanks.

**Pull request merged**